### PR TITLE
[8.0] Initial commit, implementation of the module.

### PR DESCRIPTION
**Pull request by daramousk**
The warning that was coming up when you try to reconcile bank statements that have accounts of different type is removed.

### CODE REVIEW

File: account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//-*- coding: utf-8 -*-
+//© 2017 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+openerp.account_multiple_account_type_reconciliation = function(instance) {
+	instance.web.account.bankStatementReconciliationLine.include({
+
+		selectMoveLine : function(mv_line) {
```
Comment (NL66278) [inline account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js:8]: I am missing any comments on what this function is doing, or how it implements the requested functionality.

Comment (daramousk) [reply account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js:8]: When you try to reconcile bank statements that belong to a payable account and to a receivable account (debit, credit) you get the warning (web.Dialog) on `account/static/src/js/account_widgets.js`  at around line 1057. This module basically removes this warning by overriding the `selectMoveLine` method.

Comment (NL66278) [reply account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js:8]: @daramousk Thanks for the explanation, please put it also as comments (maybe withouth the exact linenumber, just point out the method) in the code. Is there no back-end check on reconciling debit and credit, or is it all in the frontend?

Comment (daramousk) [reply account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js:8]: @NL66278 It is just in the frontend, the reconciliation will continue normally.

**Pull request closed without merging**